spring-security-modules/spring-security-web-digest-auth/src/main/java/com/baeldung/client/HttpComponentsClientHttpRequestFactoryDigestAuth.java

```diff
@@ -22,7 +23,6 @@ public HttpComponentsClientHttpRequestFactoryDigestAuth(final HttpHost host, fin
     }
 
     //
```

Comment:
Can you remove this also pls.

Reply:
I removed just the comment line. Or do you mean the entire method: `createHttpContext`?
Thanks

Reply:
Just the stray comment line. This is good. Thanks!